daemon/events/metrics.go

```diff
@@ -0,0 +1,21 @@
+package events
+
+import (
+	"github.com/docker/docker/dockerversion"
+	"github.com/docker/go-metrics"
+)
+
+var (
+	eventsCounter    metrics.Counter
+	eventSubscribers metrics.Gauge
+)
+
+func init() {
+	ns := metrics.NewNamespace("engine", "events", metrics.Labels{
+		"version": dockerversion.Version,
+		"commit":  dockerversion.GitCommit,
+	})
+	eventsCounter = ns.NewCounter("events", "The number of events logged")
```

Comment:
Counters end in _total by convention.

events_logged_total would be a better name.